### PR TITLE
Add new failing tests to the A770 skip list

### LINKED ISSUE #1689: [A770] New failing tests in the nightly run
https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10072933761/job/27845901887
```
=========================== short test summary info ============================
FAILED language/test_core.py::test_dot[1-64-128-128-4-True-True-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-4-True-True-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-4-False-True-none-tf32-int8-int8-1_0] - RuntimeError: Triton Error [ZE]: 0x70000004
FAILED language/test_core.py::test_dot[1-64-128-128-4-False-True-none-tf32-int8-int8-1_1] - RuntimeError: Triton Error [ZE]: 0x70000004
= 4 failed, 9479 passed, 22 skipped, 1258 xfailed, 168 warnings in 842.37s (0:14:02) =
```

### PR DESCRIPTION
**Pull request by pbchekin**
Fixes #1689.

### CODE REVIEW

File: scripts/skiplist/a770/language.txt [hunk 1]
```diff
@@ -472,3 +472,8 @@ test/unit/language/test_core.py::test_dot[1-128-128-64-4-False-True-none-tf32-fl
 test/unit/language/test_core.py::test_dot[1-128-128-64-4-False-True-none-tf32-float16-float16-1_1]
 test/unit/language/test_core.py::test_dot[1-128-128-64-4-False-False-none-tf32-float16-float16-1_0]
 test/unit/language/test_core.py::test_dot[1-128-128-64-4-False-False-none-tf32-float16-float16-1_1]
+# https://github.com/intel/intel-xpu-backend-for-triton/issues/1689
+test/unit/language/test_core.py::test_dot[1-64-128-128-4-True-True-none-tf32-int8-int8-1_0]
+test/unit/language/test_core.py::test_dot[1-64-128-128-4-True-True-none-tf32-int8-int8-1_1]
+test/unit/language/test_core.py::test_dot[1-64-128-128-4-False-True-none-tf32-int8-int8-1_0]
+test/unit/language/test_core.py::test_dot[1-64-128-128-4-False-True-none-tf32-int8-int8-1_1]
```
Comment (anmyachev) [inline scripts/skiplist/a770/language.txt:479]: These are the same tests that were enabled there: https://github.com/intel/intel-xpu-backend-for-triton/pull/1667/files

**Review by anmyachev — APPROVED** [review]

File: scripts/skiplist/a770/language.txt (hunk 1 above)
Comment (anmyachev) [reply scripts/skiplist/a770/language.txt:479]: The last successful one was on this commit: https://github.com/intel/intel-xpu-backend-for-triton/commit/aa12f1639882b50e5e0d71a6b5f78fe432d55946. Just before https://github.com/intel/intel-xpu-backend-for-triton/commit/a13704ed7e3a7930b139f58d27d90950feb83130.

Comment (pbchekin) [reply scripts/skiplist/a770/language.txt:479]: > The last successful one was on this commit: [aa12f16](https://github.com/intel/intel-xpu-backend-for-triton/commit/aa12f1639882b50e5e0d71a6b5f78fe432d55946). Just before [a13704e](https://github.com/intel/intel-xpu-backend-for-triton/commit/a13704ed7e3a7930b139f58d27d90950feb83130).

Right, we should not have them enabled for a770 in the first place.

**Comment by pbchekin** [comment]
Successful run: https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10082810698

**Pull request merged**